.gitignore

```diff
@@ -47,6 +47,7 @@ htmlcov/
 .coverage
 .coverage.*
 .cache
+.test.sh.swp
```

Comment:
That's not necessary, it's specific to your editor (vim I imagine) and should be set in your [global .gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)